packages/office-ui-fabric-react/src/components/Modal/Modal.styles.ts

```diff
@@ -67,7 +67,10 @@ export const getStyles = (props: IModalStyleProps): IModalStyles => {
         position: 'relative',
         textAlign: 'left',
         outline: '3px solid transparent',
-        maxHeight: '100%',
+        maxHeight: 'calc(100% - 32px)',
+        maxWidth: 'calc(100% - 32px)',
```

Comment:
would vh be a better value here, rather than calc.

Reply:
The calc is to act as a constant max space between the the Modal and the edge of the viewport (16px).